src/app/api/[[...route]]/routes/rotation.route.ts

```diff
@@ -1,14 +1,35 @@
+import { PrismaClient } from '@prisma/client';
 import { Hono } from 'hono';
 
 const app = new Hono();
+const prisma = new PrismaClient();
```

Comment:
This is to improve the performance in the development stage, but could you utilize the Prisma object from [`@/lib/prisma`](https://github.com/Tascurator/tascurator-frontend/blob/develop/src/lib/prisma.ts)?

Learn more: [Best practice for instantiating Prisma Client with Next.js](https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices)

Reply:
I modified the import source.